src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -403,6 +412,20 @@ export default {
 </script>
 
 <style lang="postcss" scoped>
+	.placeholder > div {
+		@apply tw-rounded !important;
+	}
+
+	.placeholder {
+		display: none;
+	}
+
+	@media screen and (min-width: 600px) {
+		.placeholder {
+			display: flex;
```

Comment:
These two displays could be converted to TW.

Reply:
I think the TW breakpoint is slightly different (734px) ?

Reply:
Good call, leaving as is for now.